drivers/gles3/rasterizer_gles3.h

```diff
@@ -45,14 +44,13 @@ class RasterizerGLES3 : public RendererCompositor {
 	float delta = 0;
 
 	double time_total = 0.0;
-	double time_scale = 1.0;
 
 protected:
 	RasterizerCanvasGLES3 canvas;
 	RasterizerStorageGLES3 storage;
 	RasterizerSceneGLES3 scene;
 
```

Comment:
I found if you swap the order to:
```
	RasterizerStorageGLES3 storage;
	RasterizerCanvasGLES3 canvas;
	RasterizerSceneGLES3 scene;
```
This cures the crash on exit, because you want the storage (and RID owners) to be constructed _before_ canvas and scene, and destroyed _after_ canvas and scene.